src/fonts.ts

```diff
@@ -50,8 +50,12 @@ export const generateFonts = (
   options: TypesettingOptions = { }
 ): TypesettingResults => {
   const { family, fallbacks, variants } = typesettings;
-  const fontFamily = getFontStack(family, fallbacks);
 
+  if (!variants) {
+    throw Error('Missing variants in your typesettings');
+  }
```

Comment:
Same comment as above. Don't have to throw at runtime unless you really want to.

Reply:
I think you can remove this block. The interface requires for variants to be an array: `variants: FontVariant[];`

Reply:
removed